TestUnit/src/manager/PersonTest.java

```diff
@@ -18,9 +18,9 @@
         public void test(){
         
             Person person = new Person("alina",15); 
-            Person person1 = Mockito.spy(person);
+           Person person1 = Mockito.spy(person);
```

Comment:
Try to rename the variable **person1** - could be misleading and assumed that it's a proper object and not a mock.